exo-go/src/application/deployer.go

```diff
@@ -19,12 +21,30 @@ type Deployer struct {
 // Start starts the deployment process
 func (d *Deployer) Start() error {
 	terraformDir := d.getTerraformDir()
+	terraformConfig := types.TerraformConfig{
+		AppConfig:      d.AppConfig,
+		ServiceConfigs: d.ServiceConfigs,
+		AppDir:         d.AppDir,
+		HomeDir:        d.HomeDir,
+		TerraformDir:   terraformDir,
+		RemoteBucket:   fmt.Sprintf("%s-terraform", d.AppConfig.Name),
+		LockTable:      "TerraformLocksTest",
+		Region:         "us-west-2", //TODO prompt user for this
+	}
+
+	// Initialize blank AWS account
+	err := aws.InitAccount(terraformConfig.RemoteBucket, terraformConfig.LockTable, terraformConfig.Region)
+	if err != nil {
+		return err
+	}
 
-	err := terraform.GenerateFile(d.AppConfig, d.ServiceConfigs, d.AppDir, d.HomeDir, terraformDir)
+	// Generate Terraform Scripts
+	err = terraform.GenerateFile(terraformConfig)
 	if err != nil {
 		return err
 	}
 
+	// Run Terraform commands
```

Comment:
I don't think these comments are really adding anything. Also similarly to the previous PR, I'd suggest removing the empty newlines in the functions